### PR TITLE
Sync stylo bindings

### PR DESCRIPTION
**Pull request by Manishearth**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/12499)

### CODE REVIEW

File: ports/geckolib/gecko_bindings/bindings.rs [hunk 1]
```diff
@@ -276,7 +276,7 @@ extern "C" {
     pub fn Servo_StyleSheetHasRules(sheet: *mut RawServoStyleSheet) -> bool;
     pub fn Servo_InitStyleSet() -> *mut RawServoStyleSet;
     pub fn Servo_DropStyleSet(set: *mut RawServoStyleSet);
-    pub fn Servo_ParseStyleAttribute(bytes: *const u8, length: u32,
```
Comment (timvandermeij) [inline ports/geckolib/gecko_bindings/bindings.rs]: This reverts https://github.com/servo/servo/commit/0f09a9f357c3c038ffc0b138772626f2d4256837. Is that intended?

Comment (Manishearth) [inline ports/geckolib/gecko_bindings/bindings.rs]: Yes and no. https://bugzilla.mozilla.org/show_bug.cgi?id=1287383 hasn't landed yet, and it wasn't cherry-picked onto the stylo branch when Emilio did the rebase. When that patch lands, I'll do a resync.

We want it such that the code in https://github.com/servo/gecko-dev (stylo branch) always matches the code on Servo master. Generally if a patch hasn't yet landed in Gecko (but the corresponding patch has landed in Servo), you should just cherry-pick onto the fork (which wasn't done here, or it was lost in the rebase).

I could cherry pick, however the Gecko patch should land soon anyway, after which I can do another sync.


cc @heycam

Comment (heycam) [inline ports/geckolib/gecko_bindings/bindings.rs]: I neglected to cherry pick the patch onto the gecko-dev stylo branch. I'll remember to do that next time.

Comment (Manishearth) [inline ports/geckolib/gecko_bindings/bindings.rs]: No problem, I'll do a sync when gecko-dev gets updated

Comment (Manishearth) [inline ports/geckolib/gecko_bindings/bindings.rs]: This was addressed in the latest sync

**Pull request merged**